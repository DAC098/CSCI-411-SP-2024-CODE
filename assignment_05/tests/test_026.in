512 183
2
9
68
86
86
111
118
130
140
163
181
239
271
273
287
291
302
332
340
390
431
439
441
487
497
503
541
553
573
612
623
630
635
645
652
716
728
730
746
753
771
780
785
820
838
874
875
925
930
932
1029
1038
1069
1074
1094
1161
1186
1202
1232
1234
1239
1253
1260
1295
1313
1324
1339
1401
1425
1434
1473
1520
1533
1544
1585
1598
1633
1636
1669
1697
1703
1710
1731
1740
1752
1761
1765
1796
1861
1891
1896
1901
1903
1931
1933
1963
1968
1969
1975
1980
1996
2006
2015
2017
2044
2074
2082
2131
2173
2198
2242
2304
2331
2360
2399
2403
2405
2431
2446
2457
2505
2523
2527
2538
2560
2609
2630
2639
2692
2707
2733
2749
2764
2766
2824
2845
2867
2883
2898
2907
2910
2955
2995
3000
3003
3035
3035
3060
3063
3074
3077
3093
3095
3136
3199
3210
3225
3263
3285
3287
3293
3321
3325
3334
3352
3353
3375
3406
3435
3469
3483
3531
3555
3614
3616
3625
3652
3656
3681
3690
3776
3782
3782
3794
3804
3833
3856
3898
3945
3959
4004
4017
4029
4065
4089
4089
4102
4145
4156
4186
4202
4212
4219
4220
4229
4234
4252
4257
4286
4290
4325
4332
4402
4408
4411
4421
4438
4447
4453
4467
4469
4481
4485
4486
4512
4545
4556
4589
4590
4607
4628
4642
4650
4657
4661
4666
4670
4688
4716
4724
4754
4759
4767
4781
4816
4824
4903
4904
4915
4933
4974
4978
4979
4987
4995
5074
5091
5114
5178
5201
5211
5230
5259
5316
5329
5333
5352
5368
5382
5450
5471
5484
5518
5538
5542
5550
5567
5587
5623
5628
5643
5648
5673
5681
5697
5710
5728
5742
5744
5754
5770
5790
5899
5931
5947
5958
5980
5983
5996
6002
6026
6064
6099
6154
6170
6176
6184
6193
6197
6219
6268
6269
6291
6293
6295
6297
6308
6319
6356
6378
6383
6390
6394
6421
6434
6480
6529
6535
6542
6576
6586
6589
6591
6597
6598
6647
6667
6692
6740
6764
6774
6783
6809
6819
6821
6838
6846
6885
6923
6938
6938
6939
6968
6972
6986
7025
7026
7027
7061
7067
7129
7135
7151
7160
7160
7165
7206
7243
7287
7300
7301
7351
7395
7410
7412
7479
7501
7506
7520
7552
7582
7584
7605
7630
7631
7633
7663
7671
7675
7679
7696
7701
7722
7725
7754
7765
7775
7849
7871
7877
7920
7966
8088
8114
8117
8121
8133
8133
8150
8166
8177
8177
8189
8200
8200
8203
8230
8249
8254
8275
8276
8285
8288
8303
8307
8345
8359
8396
8439
8439
8441
8468
8483
8488
8546
8582
8596
8599
8619
8619
8640
8646
8665
8669
8686
8732
8736
8740
8743
8744
8751
8753
8754
8784
8786
8842
8878
8904
8943
8958
8984
9011
9012
9078
9097
9103
9104
9108
9110
9160
9192
9209
9226
9265
9268
9321
9376
9387
9388
9405
9415
9418
9442
9454
9479
9484
9497
9518
9525
9540
9594
9616
9632
9648
9685
9694
9732
9740
9753
9768
9795
9808
9830
9851
9864
9865
9948
9951
9953
9967
9969
9992
