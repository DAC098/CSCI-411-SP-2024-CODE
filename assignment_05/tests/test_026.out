Enter the number of hotels and the ideal number of miles to travel per day: Enter 512 hotel distances each on a separate line: Hotels to visit: 11 20 29 39 48 55 63 69 76 88 100 109 114 124 131 141 153 159 170 177 186 192 202 213 229 244 253 259 268 276 288 294 303 313 326 337 347 357 367 373 382 397 402 410 427 435 446 458 464 474 484 493 502 512 
Total penalty: 4576
